### ISSUE TITLE
Humidity adjustment for NOx in Movesmrg not applied correctly

### ISSUE THREAD

**Issue by christos-e** [CEMPD/SMOKE#44]
Movesmrg was probably coded to apply the adjustment to HONO/NO/NO2 (and total NOX) but not HONO_INV/NO_INV/NO2_INV. When running with the speciation option, model-ready HONO/NO/NO2 are derived from HONO_INV/NO_INV/NO2_INV. (The EF tables contain both HONO_INV/NO_INV/NO2_INV, which have mass units, and HONO/NO/NO2, which have moles units. With the speciation option, we need to use the mass unit variables rather than the moles unit variables because SMOKE will convert to moles when it “speciates”.)

**Comment by eyth** [comment]
Yes, it was not designed to adjust the _INV species, although I thought the
nature of Chris' comment was different than this.
